ccm_web/client/src/models/canvas.ts

```diff
@@ -54,47 +56,96 @@ export class CanvasCourseSectionSort_UserCount implements ICanvasCourseSectionSo
   }
 }
 
-export enum CanvasRoleType {
+export enum CanvasEnrollmentType {
   Student = 'StudentEnrollment',
   Teacher = 'TeacherEnrollment',
   TA = 'TaEnrollment',
   Observer = 'ObserverEnrollment',
   Designer = 'DesignerEnrollment'
 }
 
-export enum ClientRoleType {
+export enum ClientEnrollmentType {
   Student = 'student',
```

Comment:
I don't 100 percent remember why I changed the names here. I don't think it's a major issue, but if someone feels strongly, I can change it back to `ClientRoleType` and `CanvasRoleType`. Maybe I was trying to distinguish between `RoleEnum` and `CanvasRoleType`. I wonder if there's a way to combine these enums, but that may be out of scope for this issue.